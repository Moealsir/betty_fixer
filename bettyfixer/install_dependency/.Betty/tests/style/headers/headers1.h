#ifndef _HEADERS1_H_
#define _HEADERS1_H_

#define sample 98
#define sample1 98 + 402

struct sample_s {
	char *str;
	int i;
};

union sample_u {
	char rgba[4];
	int value;
};

enum sample_e {
	first = 0,
	second,
	third
};

/* ----- */

typedef struct sample_s sample_s;
typedef union sample_u sample_u;
typedef enum sample_e sample_e;

/* ----- */

typedef struct sample1_s {
	char *str;
	int i;
} sample1_s;

typedef union sample1_u {
	char rgba[4];
	int value;
} sample1_u;

typedef enum sample1_e {
	first = 0,
	second,
	third
} sample1_e;

#endif /* _HEADERS1_H_ */
