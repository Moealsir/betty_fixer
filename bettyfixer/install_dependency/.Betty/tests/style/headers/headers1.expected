tests/style/headers/headers1.h:5: ERROR: Macros with complex values should be enclosed in parentheses
tests/style/headers/headers1.h:7: ERROR: open brace '{' following struct go on the next line
tests/style/headers/headers1.h:12: ERROR: open brace '{' following union go on the next line
tests/style/headers/headers1.h:17: ERROR: open brace '{' following enum go on the next line
tests/style/headers/headers1.h:31: ERROR: open brace '{' following struct go on the next line
tests/style/headers/headers1.h:36: ERROR: open brace '{' following union go on the next line
tests/style/headers/headers1.h:41: ERROR: open brace '{' following enum go on the next line
total: 7 errors, 0 warnings, 47 lines checked
